popdat/scripts/build_population_areas.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -e
+
+# Outputs a FlatGeoBuf of geometries and their corresponding population,
+# suitable for use in the popdat crate.
+#
+# You'll need to update the `configuration` section to get this to work on your
+# machine.
+# 
+# Note: I've only ever used this for US data, with "Census Blocks" as areas. In
+# theory other countries could also massage their data to fit into the combined
+# FlatGeoBuf.
+#
+# ## Input
+#
+# I wasn't able to find a reasonable solution using the official census.gov
+# files or API's. Instead I batch-downloaded all the census blocks and their
```

Comment:
Ah, nhgis.org needs a login and has a fancy interface for selecting downloads, so we can't just stick a URL in here. :(  The number of tables they have is a bit overwhelming and I couldn't easily tell what you've selected -- which year, table name? Could you be a bit more specific here so the instructions are more reproducible?

Reply:
I've added some specific steps